awx/main/tests/unit/models/test_workflow_unit.py

```diff
@@ -194,18 +194,18 @@ def test_inherit_workflow_job_extra_vars(self, job_node_no_prompts):
             extra_vars={'a': 84}, **self.kwargs_base)
 
     def test_char_prompts_and_res_node_prompts(self, job_node_with_prompts):
+        # TBD: properly handle multicred credential assignment
```

Comment:
Doing some minor hand-waving here as this related directly to some changes @AlanCoding and @jangsutsr are working on.